charts/terra-argocd-env/README.md

```diff
@@ -1,5 +1,21 @@
-# terra-argocd-env
+terra-argocd-env
+================
+A Helm chart for generating environment-wide ArgoCD resources for Terra environments
 
-This Helm chart generates an ArgoCD Project definition for a Terra environment.
+Current chart version is `0.0.7`
```

Comment:
Do we need a re-iteration of the chart version in the README? Also this points to the old version.